cmd/vic-machine/create/create.go

```diff
@@ -998,6 +1004,16 @@ func (c *Create) processRegistries() error {
 		c.InsecureRegistries = append(c.InsecureRegistries, *regurl)
 	}
 
+	// load a list of whitelisted registries
+	for _, registry := range c.whitelistRegistries {
+		regurl, err := validate.ParseURL(registry)
```

Comment:
I am not sure storing all the registries as `url.URL` objects is a good idea. In particular, storing a CIDR this way seems to more trouble that its worth. I would store them just as strings, and then have a custom `parse` function to give me back an object that represents a registry entry. You can add methods to that object/interface to check if a given ip address or hostname matches against that entry.